.github/workflows/build.yml

```diff
@@ -21,16 +21,16 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: [7.4, 7.3]
-                symfony: [^4.4, ^5.2]
-                sylius: [~1.8.0, ~1.9.0]
-                node: [10.x]
-                mysql: [5.7]
+                php: ["8.0", "7.4"]
+                symfony: ["^4.4", "5.2.*"]
+                sylius: ["~1.9.0", "~1.10.0"]
+                node: ["10.x"]
+                mysql: ["8.0"]
```

Comment:
This is a bad idea imho.

Upstream specifically pinned Symfony at `5.2.*`, they (again) just haven't updated all their code.
This was necessary because of a bug introduced by Symfony and I think testing has to reflect that.


Do you really want to test Sylius 1.10 alpha?

I understand your idea to align with the plugin, but this doesn't make sense (Sylius has been very lazy to update their code for 1.10 altogether).

Reply:
> What do you mean with "Upstream specifically pinned Symfony at 5.2.*"? What bug are you referring to? And why PluginSkeleton is green with `^5.2`. Note that `5.2.*` means that it will not be tested with Symfony 5.3 or 5.4 which are already out.

There was an interface change in Symfony 5.3 that broke 5.2 without prior notice.

Seems it was fixed in 5.3 eventually, but Sylius required 5.2.* for some time (see here https://github.com/Sylius/Sylius/pull/12682). If the tests run well I don't see a reason to require 5.2.* now.